block_designs.tex

```diff
@@ -900,4 +893,136 @@ \section{Steinerovské systémy trojíc, zovšeobecnenia}
 
 \end{theorem_hard}
 
-\TODO konečné jednoduché grupy.
+\section{Symetrické konfigurácie}
+
+\begin{definition}
+	Konfigurácia je množina bodov a priamok, píšeme $(v_\gamma l_\pi)$, kde $v$ je počet bodov, $l$ je počet priamok, $\gamma$ je počet priamok prechádzajúcich každým bodom a $\pi$ je počet bodov na každej priamke.
```

Comment:
Nechýba čiarka v zátvorke? Osobne mi to príde kus divné označenie, indexovať čísla číslami. Ale ak ste to tak mali na prednáške, tak kľudne môže byť.

Reply:
Pridal som čiarku. Je to štandardný zápis, používajú v literatúre to týmto spôsobom.